cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -39,7 +39,8 @@ func (c *CredentialsCmd) Run(context *CliContext) error {
 		c.RoleARN,
 		src,
 		c.SessionDuration,
-		context.STSProvider)
+		context.STSProvider,
+		c.STSEndpoint)
 	if err != nil {
```

Comment:
Previously we were populating the STS endpoint but never doing anything with it - now we're passing it in to where it can be used.